parachain/pallets/outbound-queue/runtime-api/README.md

```diff
@@ -0,0 +1,6 @@
+# Ethereum Outbound Queue Runtime API
+
+Provides an API:
+
+- to prove BEEFY messages
```

Comment:
To generate merkle proofs for outbound messages.

Reply:
😅 Dankie! https://github.com/Snowfork/snowbridge/pull/1101/commits/5afeb2d0a99270971cf99d94a4e65a3a39fdeccd